### PR TITLE
TAJO-2176: Refining function documents

### PR DESCRIPTION
**Pull request by eminency**
Done.

### CODE REVIEW

File: tajo-docs/src/main/sphinx/functions/agg_func.rst [hunk 1]
```diff
@@ -1,91 +1,97 @@
...
   :param expression:
-  :type expression: int | float
-  :rtype: float8
\ No newline at end of file
+  :type expression: INT* | FLOAT*
+  :rtype: FLOAT8
+
+
+.. note:: ``INT*`` means various size of integer types can be accepted. And ``FLOAT*`` means both of ``FLOAT4`` and ``FLOAT8`` are OK.
+      For your information, in Tajo SQL, ``INT`` is alias for ``INT4`` and ``FLOAT`` is one for ``FLOAT4``.
```
Comment (jihoonson) [inline tajo-docs/src/main/sphinx/functions/agg_func.rst]: ```Tajo SQL``` looks that Tajo has own SQL syntax to me. How about just ```in Tajo```?

File: tajo-docs/src/main/sphinx/functions/math_func_and_operators.rst [hunk 2]
```diff
@@ -15,78 +15,78 @@ Math Functions and Operators
...
+  :rtype: FLOAT8
   :example:
 
   .. code-block:: sql
 
     select acos(0.8); 
     > 0.9272952180016123
 
-.. function:: atan (number float8)
+.. function:: atan (number FLOAT8)
```
Comment (jihoonson) [inline tajo-docs/src/main/sphinx/functions/math_func_and_operators.rst]: need to be FLOAT*

File: tajo-docs/src/main/sphinx/functions/math_func_and_operators.rst [hunk 3]
```diff
@@ -179,115 +179,120 @@ Math Functions and Operators
...
+  :rtype: FLOAT8
   :example:
 
   .. code-block:: sql
 
     select sign(-8.4); 
     > -1.0
 
-.. function:: sin (number float)
+.. function:: SIN (number FLOAT*)
```
Comment (jihoonson) [inline tajo-docs/src/main/sphinx/functions/math_func_and_operators.rst]: The function name seems to be lowercase.

File: tajo-docs/src/main/sphinx/functions/math_func_and_operators.rst [hunk 4]
```diff
@@ -179,115 +179,120 @@ Math Functions and Operators
...
+  :rtype: FLOAT8
   :example:
 
   .. code-block:: sql
 
     select sin(1.0); 
     > 0.8414709848078965
 
-.. function:: sqrt (number float8)
+.. function:: sqrt (number FLOAT8)
```
Comment (jihoonson) [inline tajo-docs/src/main/sphinx/functions/math_func_and_operators.rst]: need to be FLOAT*

File: tajo-docs/src/main/sphinx/functions/string_func_and_operators.rst [hunk 5]
```diff
@@ -347,82 +347,82 @@ String Functions and Operators
...
+  :rtype: TEXT
   :example:
 
   .. code-block:: sql
 
     select lower('TAJO');
     > tajo
 
-.. function:: lpad(source text, number int4, pad text)
+.. function:: lpad(source TEXT, number INT4, pad TEXT)
```
Comment (jihoonson) [inline tajo-docs/src/main/sphinx/functions/string_func_and_operators.rst]: pad can be omitted.

File: tajo-docs/src/main/sphinx/functions/string_func_and_operators.rst [hunk 6]
```diff
@@ -347,82 +347,82 @@ String Functions and Operators
...
+  :rtype: TEXT
   :example:
 
   .. code-block:: sql
 
     select lpad('hi', 5, 'xy');
     > xyxhi
 
-.. function:: rpad(source text, number int4, pad text)
+.. function:: rpad(source TEXT, number INT4, pad TEXT)
```
Comment (jihoonson) [inline tajo-docs/src/main/sphinx/functions/string_func_and_operators.rst]: pad can be omitted.

**Comment by eminency** [comment]
@jihoonson Thanks for good catches! I fixed them, please check.

**Comment by jihoonson** [comment]
+1. Thank you for the quick update!

**Comment by eminency** [comment]
Thanks, I'll ship it and close the PR.

**Pull request closed without merging**